posthog-react-native/src/posthog-rn.ts

```diff
@@ -39,6 +39,9 @@ export class PostHog extends PostHogCore {
 
     // It is possible that the old library was used so we try to get the legacy distinctID
     void preloadSemiAsyncStorage().then(() => {
+      // We run this extra here as the storage may have been reset...
+      this.setupBootstrap(options)
```

Comment:
oh, interesting. I see, in react-native, storage is setup async, and we'll need this to be present before we can bootstrap FFs?

.. I guess except when using the `memory` option?

Reply:
Yeah pretty much. It's annoying but there isn't much way around it